index.html

```diff
@@ -0,0 +1,227 @@
+  </head>
+  <body>
+    <!-- Introduction -->
+    <section id="intro" class="pb-4">
+        <header>
+            <ul class="header m-0">
+                <li class="header-item"><a class="text-white" href="#"><i class="fa-brands fa-facebook"></i></a></li>
+                <li class="header-item"><a class="text-white" href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                <li class="header-item">English</li>
+                <li class="header-item">My Page</li>
+                <li class="header-item">Logout</li>
+            </ul>
+        </header>
+        <div class="container">
+            <div class="row mx-0">
+                <div class="col">
+                    <nav class="navbar">
+
+                        <a href="index.html"><img class="logo" alt="Logo" src="images/web-development-logo.jpg"></a>
+                
+                        <ul class="nav-options-ul">
+                            <li class="nav-options-ul-li"><a class="nav-options-ul-li-a" href="about.html">About</a></li>
+                            <li class="nav-options-ul-li"><a class="nav-options-ul-li-a" href="#main-program">Courses</a></li>
+                            <li class="nav-options-ul-li"><a class="nav-options-ul-li-a" href="#join">Join</a></li>
+                            <li class="nav-options-ul-li"><a class="nav-options-ul-li-a" href="#sponser">Sponser</a></li>
+                            <li class="nav-options-ul-li"><a class="nav-options-ul-li-a" href="#news">News</a></li>
+                        </ul>
+                
+                        <!-- for mobile -->
+                
+                        <div class="hamburger">
+                            <span class="bar"></span>
+                            <span class="bar"></span>
+                            <span class="bar"></span>
+                        </div>
+                
+                        <ul class="nav-menu">
+                            <li class="nav-options"><a class="nav-options-ul-li-a" href="about.html">About</a></li>
+                            <li class="nav-options"><a class="nav-options-ul-li-a" href="#main-program">Courses</a></li>
+                            <li class="nav-options"><a class="nav-options-ul-li-a" href="#join">Join</a></li>
+                            <li class="nav-options"><a class="nav-options-ul-li-a" href="#sponser">Sponser</a></li>
+                            <li class="nav-options"><a class="nav-options-ul-li-a" href="#news">News</a></li>
+                        </ul>
+            
+                    </nav>
+                </div>
+            </div>
+            <div class="row mx-0 mt-5 gy-3">
+                <div class="col-12">
+                    <p class="welcome-note fs-5 mb-0">"Hello! Sharing World"</p>
+                    <h1 class="text-uppercase fw-bold">Learn Web Development With Us</h1>
+                </div>
+                <div class="col-12">
```

Comment:
### Design and styling
Kindly remember, all page elements should have similar spacing, borders, shadows, and paddings to the original design.
```
Basic color swatch: #272A31, #EC5242, #D3D3D3
Fonts used originally: Cocogoose, Lato
```
- [ ] It would be better for all sections to increase the ```height``` and the ```spacing (padding and margin)``` to replicate the Design 👍 👍 
    - You may follow this [link](https://www.geeksforgeeks.org/css-margins-padding/) for more info on CSS spacing 
- [ ] I think you may also try to improve the Typography of the whole page to replicate the given Design, 😂 
    - Texts are small and they are not readable on both Desktop and Mobile Version 😨 It would be better to increase the ```font-size`` and Try to use the same color for your text to replicate the given Design 😂 

**Your design**
![image](https://user-images.githubusercontent.com/102757126/210799549-f64b3756-8022-4828-923d-5e42c2710c36.png)

**Given design**
![image](https://user-images.githubusercontent.com/102757126/210799901-26c470c7-f521-4f56-8207-055331ac72d0.png)